chat/src/main/java/com/github/twitch4j/chat/events/channel/IRCMessageEvent.java

```diff
@@ -105,11 +123,25 @@ public IRCMessageEvent(String rawMessage) {
      * @param rawMessage  The raw message.
      * @param botOwnerIds The bot owner ids.
      */
-	public IRCMessageEvent(String rawMessage, Collection<String> botOwnerIds) {
+	public IRCMessageEvent(String rawMessage, Map<String, String> channelIdToChannelName, Map<String, String> channelNameToChannelId, Collection<String> botOwnerIds) {
 		this.rawMessage = rawMessage;
+		this.channelIdToChannelName = channelIdToChannelName;
+		this.channelNameToChannelId = channelNameToChannelId;
```

Comment:
Is there a long-term use for saving this to a field? Currently only seeing these maps used in the constructor

Reply:
I originally had the code somewhere else only running if getChannelId/Name was called, but decided to do it in the constructor later - but forgot to move the properties on the class.